checkbox-test-plans/units/gnome-boxes.pxu

```diff
@@ -5,10 +5,29 @@ _description:
  PURPOSE:
        Check that gnome-boxes runs and works correctly
  STEPS:
-      1. 
-      2. 
-      3. 
-      4. 
+      1. Launch gnome-boxes from the desktop icon
+      2. Make sure switching themes does what is expected. For example, if you are on light theme, then with the snap open, switch to dark theme and observe the snap adjusting to the dark theme.
+      3. Make sure hamburger menu -> Help brings up the gnome-boxes Help page
+      4. Execute the Install tests. (Instructions below)
+      5. Execute launch/shutdown test by starting an existing VM and making sure it shuts down without error.
+      6. Let the VM go to sleep and make sure it wakes up and takes you to the login screen. 
+      7. Maximize the VM window and make sure it resizes to fit the screen.
+      8. With an existing VM running, make sure you can mount/unmount an ISO and that the system sees it with `df`.
+      9. Adjust the RAM/CPU resources on an existing VM and restart it. Make sure that is reflected in the running system, verified with `htop`.
+      10. Play a video and make sure the video/audio playback works as expected.
```

Comment:
It would be good to include an example video to play back so the reader doesn't have to figure out how to test that.  Perhaps assume video/audio in firefox would be sufficient.  Maybe big buck bunny in the browser?

https://videoplayback.vhx.tv/player-team-test/videos/bbb-sunflower-native-60fps-normal

Reply:
good suggestion, and big buck bunny is my go to video for testing when I need it :)